deploy/playbooks/roles/common/tasks/main.yml

```diff
@@ -1,52 +1,41 @@
 ---
-- name: "Build hosts file"
-  sudo: yes
-  template:
-    src: hosts.j2
-    dest: "/etc/hosts"
-
-- name: Set Hostname with hostname command
```

Comment:
Ah, I do remember that issue.  What are the odds we're going to change the hostname after deploying mita though?